src/components/Modals/ContactCardModal.jsx

```diff
@@ -45,84 +45,113 @@ class ContactCardModal extends React.Component {
   render() {
     const { onClose, t, id } = this.props
     const { editMode, shouldDisplayConfirmDeleteModal } = this.state
+
     return (
-      <Query query={client => client.get(DOCTYPE_CONTACTS, id)}>
-        {({ data: contact, fetchStatus }) => {
-          return (
-            <Modal
-              into="body"
-              dismissAction={onClose}
-              size="xlarge"
-              mobileFullscreen
-            >
-              {fetchStatus !== 'loaded' && <SpinnerContact size="xxlarge" />}
-              {!editMode &&
-                fetchStatus === 'loaded' && (
-                  <ContactCard
-                    contact={contact}
-                    renderHeader={children => (
-                      <ModalHeader className="u-flex u-flex-items-center u-flex-column-s u-pr-1-half-s">
-                        {children}
-                        <div>
-                          <Button
-                            type="button"
-                            theme="secondary"
-                            icon="pen"
-                            label={t('edit')}
-                            onClick={this.toggleEditMode}
-                          />
-                        </div>
-                        <div className="u-flex u-flex-row u-ml-auto u-ml-0-s">
-                          <ContactGroups contact={contact} />
-                          <ContactCardMenu
-                            deleteAction={{
-                              label: t('delete'),
-                              action: this.toggleConfirmDeleteModal
-                            }}
-                          />
-                        </div>
-                      </ModalHeader>
-                    )}
-                    renderBody={children => (
-                      <ModalContent>{children}</ModalContent>
-                    )}
-                  />
-                )}
+      <Modal into="body" dismissAction={onClose} size="xlarge" mobileFullscreen>
+        <Query query={client => client.get(DOCTYPE_CONTACTS, id)}>
```

Comment:
why did you switch `<Modal>` and `<Query>`?

Reply:
Because I want to open the Modal before since now I need to wait the resolution of 2 requests